feature-libs/customer-ticketing/components/customer-ticketing-dialog/customer-ticketing-dialog.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
```

Comment:
let's create it under ../customer-ticketing/components/shared/customer-ticketing-dialog